test/OpenSpaceToolkit/Astrodynamics/Flight/Profile.test.cpp

```diff
@@ -370,6 +381,7 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Flight_Profile, InertialPointing)
     }
 }
 
+// TBI: Should move these tests to the Transform test suite and only test interface here
```

Comment:
Can you make a GH issue and ticket for this? Or open another MR to do this?

Reply:
Will open a follow up MR when I have some time. Don't think it's worth making a GH issue.